vmdb/spec/factories/provider.rb

```diff
@@ -0,0 +1,13 @@
+FactoryGirl.define do
+  factory :foreman_provider, :class => 'ProviderForeman' do
+    ignore do
+      userid   "admin"
+      password "smartvm"
+    end
```

Comment:
I don't understand the need for this.  Can you just hardcode it below in the after(:build) and then you don't need the evaluator?